File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/how-to--add-bookmark-controls-to-word-documents_3.vb
        If Me.Application.ActiveDocument.Bookmarks.Count > 0 Then
            Dim firstBookmark As Word.Bookmark = Me.Application.ActiveDocument.Bookmarks(1)

            Dim extendedDocument As Document = Globals.Factory.GetVstoObject(Me.Application.ActiveDocument)

            Dim vstoBookmark As Bookmark = extendedDocument.Controls.AddBookmark( _
                firstBookmark, "VSTOBookmark")
        End If

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/how-to--add-bookmark-controls-to-word-documents_3.cs
            if (this.Application.ActiveDocument.Bookmarks.Count > 0)
            {
                object index = 1;
                Word.Bookmark firstBookmark = this.Application.ActiveDocument.Bookmarks.get_Item(ref index);


                 Document extendedDocument = Globals.Factory.GetVstoObject(this.Application.ActiveDocument);

                Bookmark vstoBookmark = extendedDocument.Controls.AddBookmark(
                        firstBookmark, "VSTOBookmark");
            }